CMakeLists.txt

```diff
@@ -96,7 +96,6 @@ if (USE_OLED_VERSION)
         "${INC_DIR}/lcdlib_common_oled.h"
     )
     message(STATUS "Building the OLED version of lcdlib")
-    set(LCDLIB_LIB_VERSION_MAJOR 1)  # Increment major version for new version
```

Comment:
why are we removing the Version details? may be we should increase the Major version number.

Reply:
I removed it from my initial commit because this is a manual override. I think it just takes i from the git tags correct?